.eslintrc.json

```diff
@@ -1,11 +1,12 @@
 {
   "root": true,
   "ignorePatterns": ["**/*"],
-  "plugins": ["@nrwl/nx"],
+  "plugins": ["@nrwl/nx", "unused-imports"],
```

Comment:
Already on master, came from resetting, decided to leave it in as it doesn't hurt